### PR TITLE
Tools: Simplify update-wiitdb.sh

### PR DESCRIPTION
**Pull request by leoetlino**
After getting in touch with the GameTDB maintainer about using their databases, they have decided to implement a UNIQUE parameter so we don't need to deduplicate ourselves. This simplifies the script a bit.

Thanks to lustar for their help.

### CODE REVIEW

File: Tools/update-wiitdb.sh [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=FR fr")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=IT it")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=JA ja")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=KO ko")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=NL nl")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=PT pt")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=RU ru")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=ZHCN zh_CN")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=ZHTW zh_TW")
+dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=EN en")
```
Comment (ligfx) [inline Tools/update-wiitdb.sh]: Does this not need `UNIQUE=TRUE`?

Comment (leoetlino) [inline Tools/update-wiitdb.sh]: Nope. UNIQUE means "only return non-English titles", so using it with English doesn't make sense :P

Comment (BhaaLseN) [inline Tools/update-wiitdb.sh]: Maybe you should add that as a comment, just to be sure? ;)

Comment (leoetlino) [inline Tools/update-wiitdb.sh]: Done.

Comment (ligfx) [inline Tools/update-wiitdb.sh]: Weird!

**Pull request merged**